test/testHTML.js

```diff
@@ -713,7 +713,7 @@ ${context.content.document.body.innerHTML}`,
       },
       {
         request: { params },
-        secrets: Object.assign({ HLX_SANITIZE_DOM: true }, secrets),
+        secrets: Object.assign(secrets, { SANITIZE_DOM: true }),
```

Comment:
why did you change this? you now overwrite the global `secrets` with the `Object.assign`.

eg:

```
$ node
> const a = { foo: 42 };
undefined
> Object.assign(a, { title: 'Hello, world.' });
{ foo: 42, title: 'Hello, world.' }
> a
{ foo: 42, title: 'Hello, world.' }
>
```

Reply:
I fixed it in 6f723e86f7c2a599043e015c2499a0b007418c24

Reply:
> why did you change this?

As I explained in https://github.com/adobe/helix-pipeline/pull/361/commits/7d3956ee742d905db294687b9b95d28b491686e3, the test failed the other way round. But your fix makes much more sense :)